RELEASE/scripts/autoscend/auto_quest_level_12.ash

```diff
@@ -192,10 +140,10 @@ int auto_warAdvReqCB()
 	}
 	
 	float expectedItemDropMulti = 1 + simValue("Item Drop")/100;
```

Comment:
I messed up when I wrote this code initially. simValue gives the predicted value of Item Drop if you were to maximize with the last maximizer string used. (its is not affected by multipliers like "maximize 1000 item drop" will display actual item drop and not 1000x that value).

This could stand to be replaced with code that actually simulates maximizing for item... on the other hand its not that critical, as it is it can serve as a rough estimate as is. Since farming butterflies is a rare occurance.

Reply:
Added maximization in b16fae0.

Reply:
new code looks good